lib/cisco_node_utils/cmd_ref/snmp_notification_receiver.yaml

```diff
@@ -1,52 +1,48 @@
 # snmp_notification_receiver
 ---
-_exclude: [ios_xr]
+_template:
+  nexus:
+    get_command: "show running-config snmp all"
+  ios_xr:
+    get_command: "show running-config snmp"
 
 port:
-  get_command: "show running-config snmp all"
   get_value: '/^snmp-server host %s.*version.* udp-port (\d+).*$/'
   default_value: null
 
 receivers:
-  get_command: "show running snmp all"
   get_value: '/^snmp-server host (\S+) (traps|informs)/'
   set_value: "<state> snmp-server host <ip> <type> version <version> <security> <username> <udp_port>"
   multiple:
 
 security:
-  get_command: "show running-config snmp all"
   get_value: '/^snmp-server host %s.* version 3 (auth|noauth|priv).*$/'
   default_value: null
 
 source_interface:
-  get_command: "show running-config snmp all"
+  _exclude: [ios_xr]
   get_value: '/^snmp-server host %s.* source-interface (\S+).*$/'
   set_value: "snmp-server host <ip> source-interface <source_interface> <port>"
   default_value: null
 
 type:
-  get_command: "show running-config snmp all"
   get_value: '/^snmp-server host %s (traps|informs).*$/'
   default_value: null
 
 username:
-  get_command: "show running-config snmp all"
   get_value: '/^snmp-server host %s.*version.* (\S+)$/'
   default_value: null
 
 username_with_port:
-  get_command: "show running-config snmp all"
   get_value: '/^snmp-server host %s.*version.* (\S+) udp-port/'
   default_value: null
 
 version:
-  get_command: "show running-config snmp all"
   get_value: '/^snmp-server host %s.* version (\S+).*$/'
   default_value: null
 
 vrf:
-  get_command: "show running-config snmp all"
+  _exclude: [ios_xr]
```

Comment:
Again, why exclude on XR?

http://www.cisco.com/en/US/docs/ios_xr_sw/iosxr_r3.7/system_management/command/reference/yr37snmp.html#wp1151429

- Search for `snmp-server vrf`